src/Take.Blip.Builder/Storage/Redis/RedisNamedSemaphore.cs

```diff
@@ -1,65 +0,0 @@
-﻿using System;
```

Comment:
since you deleted RedisNamedShemaphore, why not delete MemoryNamedSemaphore?

Reply:
Because this is the only implementation of `INamedSemaphore` on this repo, and this interface is still used on `FlowManager`.